src/StatusQ/Components/StatusChart.qml

```diff
@@ -0,0 +1,133 @@
+import QtQuick 2.13
+import "private/chart/Chart.js" as Chart
+
+Canvas {
+    id: root
+
+    property var jsChart: undefined
```

Comment:
sometimes var properties are marked explicitly (like here) but in other places `undefined` is just skipped. Could we be more consistent about that?